lxd/device/cdi/id.go

```diff
@@ -0,0 +1,114 @@
+package cdi
+
+import (
+	"fmt"
+	"strings"
+
+	"tags.cncf.io/container-device-interface/pkg/parser"
+)
+
+// Vendor represents the compatible CDI vendor.
+type Vendor string
+
+const (
+	// Nvidia represents the Nvidia CDI vendor.
+	Nvidia Vendor = "nvidia.com"
+)
+
+// ToVendor converts a string to a CDI vendor.
+func ToVendor(vendor string) (Vendor, error) {
+	switch vendor {
+	case string(Nvidia):
+		return Nvidia, nil
+	default:
+		return "", fmt.Errorf("invalid CDI vendor (%q)", vendor)
```

Comment:
Errors should start with capital letter please

Reply:
fixed